.github/workflows/linux-tests-spyder-versions.yml

```
@@ -0,0 +1,66 @@
+name: Spyder Versions
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  linux:
+    name: Linux Py37 Spyder Version
+    runs-on: ubuntu-latest
+    env:
+      CI: True
+      SPYDER_VERSION: ${{ matrix.SPYDER_VERSION }}
+      RUNNER_OS: 'ubuntu'
+    strategy:
+      fail-fast: false 
+      matrix:
+        SPYDER_VERSION: ['5.1.5', '5.2.0', '5.3.0']
```

Update Spyder version to the latest patches:
```suggestion
        SPYDER_VERSION: ['5.1.5', '5.2.2', '5.3.1']
```